src/lib/integration_test_local_engine/mina_docker.ml

```diff
@@ -1091,7 +1096,20 @@ module Network_manager = struct
     let%bind () = File_system.remove_dir t.docker_dir in
     Deferred.unit
 
-  let destroy t =
+  let tear_down t network =
```

Comment:
This function should be called `generate_code_coverage` or similar, and should be called from inside the `destroy` above.